repo-client/src/main/java/io/lionweb/repoclient/api/ClassifierKey.java

```diff
@@ -0,0 +1,47 @@
+package io.lionweb.repoclient.api;
+
+import java.util.Objects;
+
+public final class ClassifierKey {
```

Comment:
I'd expect to also have something like a `LanguageKey`, as a language is identified by the tuple _(key, version)_.

Reply:
The shape of this object depends on the values returned by the LionWeb repository, so we cannot change it here without first changing it in the repository